src/HonzaBotner.Discord/Command/Permissions/PermissionResolver.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using HonzaBotner.Discord.Command.Permissions.Entities;
+using HonzaBotner.Discord.Command.Permissions.Repositories;
+
+namespace HonzaBotner.Discord.Command.Permissions
+{
+    public class PermissionResolver
```

Comment:
Do you expect that somebody may inherit this class? If not make it sealed

Reply:
My bad, in Kotlin classes are final by default and you must explicitly define them as open. I forget that this is not the case in C# :smile: